src/Tasks/TestConnectionTask.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace PhpTek\Sentry\Tasks;
+
+use Sentry\Severity;
+use SilverStripe\Dev\BuildTask;
+use function Sentry\captureMessage;
+
+class TestConnectionTask extends BuildTask
+{
+    protected $title = 'Test Sentry Configuration';
+
+    protected $description = 'Captures message for all levels available';
+
+    /**
+     * Implement this method in the task subclass to
+     * execute via the TaskRunner
+     *
+     * @param \SilverStripe\Control\HTTPRequest|null $request
+     */
+    public function run($request = null)
+    {
+        foreach( Severity::ALLOWED_SEVERITIES as $severity) {
+            $response = captureMessage('Testing Severity Level: ' . $severity, new Severity($severity));
```

Comment:
You can use [`NullTransport`](https://docs.sentry.io/platforms/php/#nulltransport) to have Sentry act like it sent events w/o actually sending them.

Reply:
I've just realised that this is using Sentry's SDK directly. Shouldn't we be testing Sentry connections through the very same system that will report errors - the module logic itself?

I can make this change another time, it isn't a biggie. 

I've merged the 2.0 PR. Once the same changes are applied to this, I'll merge.

Reply:
@phptek cool. My aim was to test the connection, but I don't mind if it's done in a different way.

Reply:
No worries, I don't want to burden you with anything further at this stage - I really appreciate the help :-) once the changes you guys have made to his task for the 2.0 release have bene made to this PR, I'm happy to merge.